prisma/schema.prisma

```diff
@@ -70,3 +70,11 @@ model Movie {
   genre String
   duration String
 }
+
+model MovieRelation {
+  id String @id @default(auto()) @map("_id") @db.ObjectId
+  primaryMovieId String
```

Comment:
it seems that Prisma doesn't support many-to-many relations within single table, thus there are no actual 'keys' to Movie. when tried to recreate that relationship as it would work with two tables I kept getting errors about missing or ambiguous "other side of the relation"